Activities/ManagePersonsPermissionCheckerTask.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.Extensions.Localization;
```

Comment:
Note what was also linked from under the issue: https://github.com/Lombiq/Orchard-Training-Demo-Module#contributing-and-support "When adding new tutorials please keep in mind the following".